CSharp/DevVmPowershell/Helpers/Interfaces/IRelativityVersionHelper.cs

```diff
@@ -8,6 +8,6 @@ namespace Helpers.Interfaces
 {
 	public interface IRelativityVersionHelper
 	{
-		void ConfirmInstallerAndInstanceRelativityVersionAreEqual(string installerRelativityVersion);
+		Task ConfirmInstallerAndInstanceRelativityVersionAreEqual(string installerRelativityVersion);
```

Comment:
Don't want to be that guy, but should this function now end in Async?

Reply:
Done